components/ai_chat/core/browser/constants.cc

```diff
@@ -40,7 +40,6 @@ base::span<const webui::LocalizedString> GetLocalizedStrings() {
       {"modelPremiumLabelNonPremium",
        IDS_CHAT_UI_MODEL_PREMIUM_LABEL_NON_PREMIUM},
       {"modelCategory-chat", IDS_CHAT_UI_MODEL_CATEGORY_CHAT},
-      {"menuNewChat", IDS_CHAT_UI_MENU_NEW_CHAT},
```

Comment:
We can't remove this string - it's still used by `error_conversation_end.tsx`

Reply:
It seems `error_conversation_end.tsx` uses the locale from `src/brave/components/ai_chat/resources/page/stories/locale.ts`, that's why I removed it here.
@petemill

Reply:
Okay, Got it. Actually I thought `error_conversation_end.tsx` only uses locale from here https://github.com/brave/brave-core/blob/9303bae9fe07bc2ec7c0cdef139e87304b2d6c96/components/ai_chat/resources/page/stories/locale.ts#L28.

Is it necessary to have these entries in two different files? Why can't we just maintain it in `.grd` file?

@petemill

Reply:
Just now I seen that it is under `stories` directory :grin: , so probably used only by storybook :+1: